metricbeat/module/logstash/_meta/supported-versions.yml

```diff
@@ -0,0 +1,5 @@
+variants:
+  - LOGSTASH_VERSION: 8.1.0-aa69d697-SNAPSHOT
+  - LOGSTASH_VERSION: 8.0.0-rc2
+  - LOGSTASH_VERSION: 7.17.1-aa300c4a-SNAPSHOT
+  - LOGSTASH_VERSION: 7.17.0
```

Comment:
I like to see that we test multiple variants. Lets make sure we do the same for the other stack modules.

Reply:
Cool. That's the plan I laid out in https://github.com/elastic/beats/issues/30146